### PR TITLE
FIX Use correct versioned state badge designs in campaign admin list items

### PR DESCRIPTION
**Pull request by robbieaverill**
Fixes https://github.com/silverstripe/silverstripe-campaign-admin/issues/77

Requires https://github.com/silverstripe/silverstripe-admin/pull/861 first.

Example screenshot:

image

The "No changes" badge used to be invisible before hovering over it. I haven't carried that change through. Would be nice to run through this with @clarkepaul in person when you're available some time. The colours are in https://github.com/silverstripe/silverstripe-admin/pull/861 and are just `$orange` and `$green` variables in admin.

### CODE REVIEW

File: client/src/containers/CampaignAdmin/CampaignAdminItem.js [hunk 1]
```diff
@@ -121,20 +124,20 @@ class CampaignAdminItem extends Component {
...
           break;
         case 'modified':
-          badge.className = 'badge badge-warning list-group-item__status';
+          badge.status = 'modified';
           badge.Title = i18n._t('CampaignAdmin.MODIFIED', 'Modified');
           break;
         case 'deleted':
-          badge.className = 'badge badge-error list-group-item__status';
+          badge.status = 'removed';
           badge.Title = i18n._t('CampaignAdmin.REMOVED', 'Removed');
```
Comment (robbieaverill) [inline client/src/containers/CampaignAdmin/CampaignAdminItem.js]: I couldn't see a way of reproducing this via the UI. When I remove a page from a campaign is just removes it from the items list.

**Comment by robbieaverill** [comment]
This will need to be updated since the admin PR has changed a bit

**Comment by emteknetnz** [comment]
This pull request hasn't had any activity for a while. Are you going to be doing further work on it, or would you prefer to close it now?

**Comment by robbieaverill** [comment]
I can take a look for sure. I seem to remember someone else in the team had made the same change anyway. I'll check tomorrow.

**Pull request closed without merging**